loryblu/loryblu/Views/LoginView/LoginView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  LoginView.swift
+//  LoryBlu
+//
+//  Created by Paulo Pinheiro on 7/25/23.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    
+    
+    func emailValidator(_ email: String) -> Bool {
+        if email.count > 100 {
+            return false
+        }
+        let emailFormat = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
+        let emailPredicate = NSPredicate(format:"SELF MATCHES %@", emailFormat)
+        return emailPredicate.evaluate(with: email)
+    }
+    
+    func passwordValidator(_ senha: String) -> Bool {
```

Comment:
Essa função não é necessária, pq aqui precisamos fazer essa verificação que seria diretamente no backend, no caso poderia ser uma informação em mock, para verificação se existe ou não esse dado no banco ou em mock. Se o email for valido a senha é inserida e verificar se o email esta cadastrado.